raysort/sort_utils.py

```diff
@@ -227,11 +232,15 @@ def generate_input(cfg: AppConfig):
                 )
                 offset += size
     logging.info("Generating %d partitions", len(tasks))
-    parts = ray.get(tasks)
+
+    tasks = ray.get(tasks)
+    parts = [p[0] for p in tasks]
+    checksums = [p[1] for p in tasks]
```

Comment:
```suggestion
    parts, checksums = zip(*ray.get(tasks))
```
I think this could be more concise, but will leave it up to @franklsf95.

Reply:
Yeah this could work. See above comment about just putting the checksum in PartInfo.